nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/ui/common/property-table/property-table.component.ts

```diff
@@ -114,6 +114,8 @@ export class PropertyTable implements AfterViewInit, ControlValueAccessor {
     @Input() goToService!: (serviceId: string) => void;
     @Input() supportsSensitiveDynamicProperties = false;
     @Input() propertyHistory: ComponentHistory | undefined;
+    @Input() supportsParameters: boolean = true;
+    @Input() forReferencedAttributes: boolean = false;
```

Comment:
I don't think the changes introduced here for the `forReferencedAttributes` Input are being used. Assuming that is the case, we should be able to revert the changes in the `PropertyTable` component.